AWS-Transcription.py

```diff
@@ -1,86 +1,102 @@
-from __future__ import print_function
+#!usr/bin/env python3
+
 import boto3
 import time
-import os
+from distutils.util import strtobool
+from pathlib import Path
 
 #Sets variables
 s3 = boto3.client('s3')
-print('Enter the s3 bucket name')
-s3Bucket = input()
```

Comment:
There's no reason to have an empty `input` with a `print` on top of it, because the former is enough to do both.